server/meta.jsx

```diff
@@ -91,17 +91,31 @@ function validateWebSDKUrl({ pathname, query }) {
     );
   }
   // check for extraneous parameters
-  Object.keys(query).forEach((param) => {
-    if (param !== "version" && param !== "origin") {
+  const validWebSDKBridgeParams = ["origin", "version", "payment-flow"];
```

Comment:
should we add 'debug' here?

Reply:
Good point @dtjones404! I'll send a follow up PR for adding support for the optional `debug` param.

Reply:
Good point @dtjones404! I'll send a follow up PR for adding support for the optional `debug` param.

Reply:
@dtjones404 here's the follow up PR to add support for the `debug` query param: https://github.com/paypal/paypal-sdk-client/pull/219